packages/components/src/Chips/InternalChipDismissible/InternalChipDismissibleInput.tsx

```diff
@@ -98,6 +98,7 @@ export function InternalChipDismissibleInput(props: ChipDismissibleInputProps) {
             role="listbox"
             id={menuId}
             className={styles.menuList}
+            data-testid="menu"
```

Comment:
Name this `chip-menu` because the chip test isn't the only test it's going to show up, It's also gonna show up on every test that implements `<Chips />`. Imagine adding a testid that is super generic that tests in jobber online uses them. You'd break all those test 😆 .

<img width="460" alt="image" src="https://user-images.githubusercontent.com/15986172/161777269-a4256f3c-9cbc-4fd6-845e-bdd08d86a3df.png">

Reply:
hahaha good point, that makes sense!